qiskit_nature/algorithms/initial_points/initial_point.py

```diff
@@ -30,27 +30,31 @@ class InitialPoint(ABC):
     """
 
     def __init__(self):
-        self._grouped_property: GroupedSecondQuantizedProperty | None = None
         self._ansatz: UCC | None = None
+        self._grouped_property: GroupedSecondQuantizedProperty | None = None
+
+    def __len__(self) -> int:
+        """The length of the computed initial point array."""
+        raise NotImplementedError
 
     @property
     @abstractmethod
-    def grouped_property(self) -> GroupedSecondQuantizedProperty:
-        """The grouped property."""
+    def ansatz(self) -> UCC | None:
+        """The UCC ansatz."""
```

Comment:
For the VSCF-case this is actually UVCC..
I'm wondering if this needs to be restricted at all in this way :thinking: